config.json

```diff
@@ -151,6 +151,17 @@
         "transforming"
       ]
     },
+    {
+      "slug": "circular-buffer",
+      "uuid": "657ac368-9b17-4f87-b815-decfe2bc0b5d",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 4,
```

Comment:
@ErikSchierboom was the difficulty change intentional here? (It was 5 previously)

Reply:
Yes it was intentional, based on the students submissions I see.